tephra-core/src/main/java/co/cask/tephra/TxConstants.java

```diff
@@ -83,6 +86,13 @@
     public static final String CFG_TX_TIMEOUT = "data.tx.timeout";
     /** Default value for transaction timeout, in seconds. */
     public static final int DEFAULT_TX_TIMEOUT = 30;
+    /**
+     * The timeout for a long running transaction, in seconds. If the transaction is not finished in that time,
+     * it is marked invalid.
+     */
+    public static final String CFG_TX_LONG_TIMEOUT = "data.tx.long.timeout";
```

Comment:
Please add this along with a description to the table of config properties in README.rst.